presto-docs/src/main/sphinx/language/time.rst

```diff
@@ -0,0 +1,57 @@
+========================
+Legacy And New Timestamp
+========================
+
+New ``TIMESTAMP`` and ``TIME`` semantics align the types with the SQL standard.
+See the following sections for details.
+
+.. note::
+
+   The new ``TIMESTAMP`` semantics is still experimental. It's recommended to keep
+   the legacy ``TIMESTAMP`` semantics enabled. You can experiment with the new semantics
+   by enabling in the global configuration or session level. The legacy semantics
```

Comment:
Let's avoid using "enable" here for the new semantics since the config property is inverted. How about

> by configuring them globally or on a per-session basis